src/main/resources/databasemigrations/V9__legg_til_status_ferdig_felt.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE status
+ADD COLUMN ferdig BOOLEAN NOT NULL DEFAULT true;
+
+
+UPDATE status
+SET ferdig = false
+WHERE id IN (
+    SELECT id
+    FROM STATUS s INNER JOIN
+         (
+             SELECT max(s_inner.TIDSPUNKT) tidspunkt, s_inner.AKTOR_ID
+             FROM STATUS s_inner
+             GROUP BY s_inner.AKTOR_ID
+         ) as s2 on s2.TIDSPUNKT = s.TIDSPUNKT AND s2.AKTOR_ID = s.AKTOR_ID
+)
```

Comment:
Forstår jeg dette riktig; sett `ferdig = false` på siste record for hver aktørId?

Reply:
Det er det som er meningen, er du enig i at det skjer? Jeg er litt bekymret for om det kan eksistere duplikater.

Reply:
Vi kunne evt tatt `MAX(ID)` per aktør

Reply:
Jeg er enig, det er i hvert fall sånn jeg leste det.  Jeg tror det er lite sannsynlig at vi finner en to records med samme tidspunkt for samme aktørId, men å bruke `MAX(ID)` kan være litt mer bulletproof.

Reply:
Jeg sendte link til Einar. Jeg er bare usikker på om det noen gang kan være at det faktisk ikke er den siste raden i databasen som er den aktive for en aktør. At en eldre rad kan være den som skal være aktiv. Kan ikke på stående fot se at det skal kunne være tilfelle men..

Reply:
Jeg tror at vi opprinnelig tenkte at timestampene skulle styre hvilken som var siste. Men det er jo heller ikke helt sikkert, kan for eksempel være forskjellig klokke på PTO-produsent og cv-api-produsent, og dermed kan vi også få meldinger hulter til bulter om vi bruker det også.

Så jeg tror max(id) er like bulletproof som noe annet